eslint.config.js

```
@@ -89,22 +89,64 @@ export default tseslint.config(
+      // svelte/*, @sveltejs/*, @juspay/* (component libraries)
+      'no-restricted-imports': [
+        'error',
+        {
+          patterns: [
+            {
+              group: ['$generated/*'],
+              message: 'Import types from $lib/types instead of $generated/.',
+            },
+          ],
+        },
+      ],
```

ESLint

[error] Expected "no-restricted-imports" to come before "prefer-template".

(perfectionist/sort-objects)